### PR TITLE
Add New Channel to V3 publishing doc

### PR DESCRIPTION
**Pull request by epananth**
Doc for adding new channel to V3

### CODE REVIEW

File: Documentation/CorePackages/Publishing.md [hunk 1]
```diff
@@ -491,6 +491,45 @@ The publishing logs are stored inside an Azure DevOps artifacts container named
 
 Under the `Publish Using Darc` job get the link to the newly queued build in the [Maestro promotion pipeline](https://dnceng.visualstudio.com/internal/_build?definitionId=750). The publishing logs are stored inside an Azure DevOps artifacts container named `PostBuildLogs`. 
 
+### How to add a new channel to use V3 publishing?
+
+In the Microsoft.DotNet.Build.Task.Feed/src/Model/PublishingConstants.cs file, create a new TargetChannelConfig
```
Comment (riarenas) [inline Documentation/CorePackages/Publishing.md:496]: Perhaps this section should go after creating the channel or verifying it exists

File: Documentation/CorePackages/Publishing.md [hunk 2]
```diff
@@ -491,6 +491,45 @@ The publishing logs are stored inside an Azure DevOps artifacts container named
...
+| isInternal  | Publishing to an internal Channel or public channel | true or false  |
+| PublishingInfraVersion | Which version of the publishing infra can use this configuration. | Enum = All(0), Legacy(1), Latest(2), Next(3)  |
+| AkaMSChannelName | The name that should be used for creating Aka.ms links for this channel.  |   |
+| ShippingFeed | The URL (including the index.json suffix) of the *shipping* feed to be used for this channel. |   | 
+| TransportFeed | The URL (including the index.json suffix) of the *transport* feed to be used for this channel. |   | 
+| SymbolsFeed | The URL (including the index.json suffix) of the *symbols* feed to be used for this channel. |   | 
+| ChecksumsFeed | The URL (including the index.json suffix) where *checksums* should be published to. | FeedForChecksums for public channel, FeedInternalForChecksums for internal  | 
+| InstallersFeed | The URL (including the index.json suffix) where *installers* should be published to. | FeedForInstallers for public channel, FeedInternalForInstallers for internal channel   |
+| SymbolTargetType | Publish to MSDL or SymWeb symbol server | PublicAndInternalSymbolTargets or InternalSymbolTargets |
```
Comment (riarenas) [inline Documentation/CorePackages/Publishing.md:513]: Maybe explain when to use either of the values?

File: Documentation/CorePackages/Publishing.md [hunk 3]
```diff
@@ -491,6 +491,45 @@ The publishing logs are stored inside an Azure DevOps artifacts container named
...
+
+```
```
Comment (riarenas) [inline Documentation/CorePackages/Publishing.md:515]: [nit]  for some syntax highlighting 
```suggestion
```C#
```

**Review by riarenas — APPROVED** [review]
Lgtm

File: Documentation/CorePackages/Publishing.md (hunk 2 above)
Comment (epananth) [reply Documentation/CorePackages/Publishing.md]: Done

File: Documentation/CorePackages/Publishing.md (hunk 1 above)
Comment (epananth) [reply Documentation/CorePackages/Publishing.md]: done

File: Documentation/CorePackages/Publishing.md [hunk 4]
```diff
@@ -491,6 +491,45 @@ The publishing logs are stored inside an Azure DevOps artifacts container named
...
+                false,
+                PublishingInfraVersion.All,
+                "generaltesting",
+                "https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json,
+                "https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json,
+                "https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing-symbols/nuget/v3/index.json",
+                FeedForChecksums,
+                FeedForInstallers,
+                PublicAndInternalSymbolTargets)
+```C#
```
Comment (riarenas) [inline Documentation/CorePackages/Publishing.md:530]: the C# marker should go on the opening set of  ```

\```C#
var highlighted = true
\```
renders as 

```C#
var highlighted = true
```

Comment (epananth) [reply Documentation/CorePackages/Publishing.md:530]: I messed up the entire block.. Thank you for spotting this !!!

**Pull request merged**